core/src/main/resources/hudson/cli/Messages.properties

```diff
@@ -81,6 +81,5 @@ Cannot build {0} because its configuration has not been saved.
 BuildCommand.CLICause.CannotBuildUnknownReasons=\
 Cannot build {0} for unknown reasons.
 
-DeleteNodeCommand.ShortDescription=\
- Deletes node(s)
+DeleteNodeCommand.ShortDescription=Deletes node(s)
```

Comment:
Could you describe what is the difference? Because previous line also has new lined message.